db/structure.sql

```diff
@@ -44,7 +48,7 @@ SET default_tablespace = '';
 SET default_with_oids = false;
 
 --
--- Name: analyses; Type: TABLE; Schema: public; Owner: -; Tablespace: 
+-- Name: analyses; Type: TABLE; Schema: public; Owner: -
```

Comment:
This is in a comment, so not super important, but do you know why "; Tablespace:" got dropped from every comment?